templates/github/.github/workflows/release.yml.j2

```diff
@@ -127,30 +176,10 @@ jobs:
         env:
           SECRETS_CONTEXT: {{ "${{ toJson(secrets) }}" }}
 
-      {%- if deploy_to_pypi %}
-      - name: Deploy plugin to pypi
-        run: bash .ci/scripts/publish_plugin_pypi.sh
-      {%- endif %}
-
-      {%- if deploy_client_to_rubygems %}
-      - name: Publish client to rubygems
-        run: bash .ci/scripts/publish_client_gem.sh
-      {%- endif %}
-
-      {%- if deploy_client_to_pypi %}
-      - name: Publish client to pypi
-        run: bash .ci/scripts/publish_client_pypi.sh
-      {%- endif %}
-
-      {%- if publish_docs_to_pulpprojectdotorg %}
-      - name: Publish docs to pulpproject.org
-        run: .ci/scripts/publish_docs.sh tag ${GITHUB_REF##*/}
-      {%- endif %}
-
-      {%- if update_redmine %}
-      - name: Update Redmine
-        run: bash .ci/scripts/update_redmine.sh
-      {%- endif %}
```

Comment:
The "if"s from here should be represented in the matrix above, IMHO.

Reply:
Oh yea, I totally forgot. Thanks.